owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/MainFileListFragment.kt

```diff
@@ -474,8 +475,8 @@ class MainFileListFragment : Fragment(),
                                 }
 
                                 FileMenuOption.REMOVE -> {
-                                    val dialogRemove = RemoveFilesDialogFragment.newInstance(file)
-                                    dialogRemove.show(requireActivity().supportFragmentManager, ConfirmationDialogFragment.FTAG_CONFIRMATION)
+                                    filesToRemove = listOf(file)
+                                    fileOperationsViewModel.showRemoveDialog(filesToRemove)
```

Comment:
Here we're working with a single file, we don't need to call the new use case, we can keep the previous code, right?

Reply:
Yes, but a file can also be a folder which is where we need to run our usecase :-)

Reply:
Aaaah, true!! Good point 🏅